packages/eslint-plugin/src/rules/local-cx-xcss/__tests__/rule.test.ts

```diff
@@ -0,0 +1,94 @@
+import { typeScriptTester as tester } from '../../../test-utils';
+import { localCXXCSSRule } from '../index';
+
+tester.run('local-cx-xcss', localCXXCSSRule, {
+  valid: [
+    `
+    import { cx } from '@compiled/react';
+
+    <Component xcss={cx({})} />
+  `,
+    `
+    import { cx } from '@compiled/react';
+
+    <Component innerXcss={cx({})} />
+  `,
+    `
+    import { cx } from '@compiled/react';
```

Comment:
The `cssMap` import should be added for completeness

Reply:
Thanks